domains

```diff
@@ -6,6 +6,16 @@
 # wildcard match (domain and all its multi-level subdomains) CNAMEs
 # against the domains in this list.
 
+# Pardot
+go.pardot.com
```

Comment:
If NextDNS does support wildcard blocking, there should be no reason not to do this here.

The entire domain is used for [Tracking](https://www.mypdns.org/w/trackware/) & [Adware](https://www.mypdns.org/w/adware/)

See my full RPZ history here: <https://www.mypdns.org/T3868>